packages/components/src/gridicon/index.tsx

```diff
@@ -43,7 +78,7 @@ const Gridicon = React.memo(
 				{ ...otherProps }
 			>
 				{ title && <title>{ title }</title> }
-				<use xlinkHref={ `${ spritePath }#${ iconName }` } />
+				{ proxiedSpritePath && <use xlinkHref={ `${ proxiedSpritePath }#${ iconName }` } /> }
```

Comment:
What does the component show during the load? An empty sizexsize box? The important thing is that the empty SVG box has the right size, so that the load doesn't cause layout shifts.

Reply:
The `svg` has it's `viewBox` and width and height set. Should be fine.